crates/trigger/src/cli.rs

```diff
@@ -165,7 +155,8 @@ where
         let mut builder = TriggerExecutorBuilder::new(loader);
         self.update_wasmtime_config(builder.wasmtime_config_mut())?;
 
-        let logging_hooks = StdioLoggingTriggerHooks::new(self.follow_components(), self.log_dir());
+        let logging_hooks =
+            StdioLoggingTriggerHooks::new(self.follow_components(), self.log.clone());
```

Comment:
Is the change from `self.log_dir()` to `self.log` correct here?  I would have thought we wanted to use the new log directory logic, but maybe that is now addressed somewhere else?